recipe_sha: 58179d769e41c88f6906267b814a922be99028500b080706e1c976a986b20ac3  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |2-
     INFO (biobambam,bin/bamfiltermc): Needed DSO lib/libstdc.so.6 found in conda-forge/linux-aarch64::libstdcxx==14.2.0=h3f4de04_1
  WARNING (biobambam,bin/bamfiltermc): Needed DSO lib/libgomp.so.1 found in ['conda-forge/linux-aarch64::_openmp_mutex==4.5=2_gnu']
  WARNING (biobambam,bin/bamfiltermc): .. but ['conda-forge/linux-aarch64::_openmp_mutex==4.5=2_gnu'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
     INFO (biobambam,bin/bamfiltermc): Needed DSO lib/libgcc_s.so.1 found in conda-forge/linux-aarch64::libgcc==14.2.0=he277a41_1
     INFO (biobambam,bin/bamfiltermc): Needed DSO aarch64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge/noarch::sysroot_linux-aarch64==2.17=h5b4a56d_18
     INFO (biobambam,bin/bamfiltermc): Needed DSO aarch64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-aarch64==2.17=h5b4a56d_18
     INFO (biobambam,bin/bamfiltermc): Needed DSO aarch64-conda-linux-gnu/sysroot/lib64/ld-linux-aarch64.so.1 found in CDT/compiler package conda-forge/noarch::sysroot_linux-aarch64==2.17=h5b4a56d_18
  WARNING (biobambam): run-exports library package conda-forge/linux-aarch64::gmp==6.3.0=h0a1ffab_2 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to build/ignore_run_exports)
  WARNING (biobambam): run-exports library package conda-forge/linux-aarch64::xerces-c==3.2.5=h595f43b_2 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to build/ignore_run_exports)
  Fixing permissions
  Packaged license file/s.
  INFO :: Time taken to mark (prefix)
          0 replacements in 0 files was 0.05 seconds
  WARNING: Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
  TEST START: /opt/conda/conda-bld/linux-aarch64/biobambam-2.0.185-h7d28f73_1.tar.bz2
  Renaming work directory '/opt/conda/conda-bld/biobambam_1734549481365/work' to '/opt/conda/conda-bld/biobambam_1734549481365/work_moved_biobambam-2.0.185-h7d28f73_1_linux-aarch64'
  shutil.move(work)=/opt/conda/conda-bld/biobambam_1734549481365/work, dest=/opt/conda/conda-bld/biobambam_1734549481365/work_moved_biobambam-2.0.185-h7d28f73_1_linux-aarch64)
  Reloading output folder (local): ...working... done
  Solving environment (_test_env): ...working... done

  ## Package Plan ##

    environment location: /opt/conda/conda-bld/biobambam_1734549481365/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol


  The following NEW packages will be INSTALLED:

      _openmp_mutex:    4.5-2_gnu               conda-forge
      biobambam:        2.0.185-h7d28f73_1      local
      boost-cpp:        1.85.0-hdad291f_4       conda-forge
      bzip2:            1.0.8-h68df207_7        conda-forge
      c-ares:           1.34.4-h86ecc28_0       conda-forge
      ca-certificates:  2024.12.14-hcefe29a_0   conda-forge
      gmp:              6.3.0-h0a1ffab_2        conda-forge
      icu:              75.1-hf9b3779_0         conda-forge
      keyutils:         1.6.1-h4e544f5_0        conda-forge
      krb5:             1.21.3-h50a48e9_0       conda-forge
      libboost:         1.85.0-h9fa81b4_4       conda-forge
      libboost-devel:   1.85.0-h37bb5a9_4       conda-forge
      libboost-headers: 1.85.0-h8af1aa0_4       conda-forge
      libcurl:          8.11.1-h6702fde_0       conda-forge
      libdeflate:       1.22-h86ecc28_0         conda-forge
      libedit:          3.1.20191231-he28a2e2_2 conda-forge
      libev:            4.33-h31becfc_2         conda-forge
      libgcc:           14.2.0-he277a41_1       conda-forge
      libgcc-ng:        14.2.0-he9431aa_1       conda-forge
      libgomp:          14.2.0-he277a41_1       conda-forge
      liblzma:          5.6.3-h86ecc28_1        conda-forge
      liblzma-devel:    5.6.3-h86ecc28_1        conda-forge
      libmaus2:         2.0.813-h3f813d2_1      bioconda
      libnghttp2:       1.64.0-hc8609a4_0       conda-forge
      libnsl:           2.0.1-h31becfc_0        conda-forge
      libssh2:          1.11.1-ha41c0db_0       conda-forge
      libstdcxx:        14.2.0-h3f4de04_1       conda-forge
      libstdcxx-ng:     14.2.0-hf1166c9_1       conda-forge
      libzlib:          1.3.1-h86ecc28_2        conda-forge
      ncurses:          6.5-hcccb83c_1          conda-forge
      nettle:           3.9.1-h9d1147b_0        conda-forge
      openssl:          3.4.0-h86ecc28_0        conda-forge
      snappy:           1.1.8-h54f1f3f_3        conda-forge
      staden_io_lib:    1.15.0-h346a1c6_3       bioconda
      xerces-c:         3.2.5-h595f43b_2        conda-forge
      xz:               5.6.3-h2dbfc1b_1        conda-forge
      xz-gpl-tools:     5.6.3-h2dbfc1b_1        conda-forge
      xz-tools:         5.6.3-h86ecc28_1        conda-forge
      zstd:             1.5.6-h02f22dd_0        conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  export PREFIX=/opt/conda/conda-bld/biobambam_1734549481365/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol
  export SRC_DIR=/opt/conda/conda-bld/biobambam_1734549481365/test_tmp
   bamsormadup --help
  bamsormadup: error while loading shared libraries: libstaden-read.so.14: cannot open shared object file: No such file or directory
  WARNING: Tests failed for biobambam-2.0.185-h7d28f73_1.tar.bz2 - moving package to /opt/conda/conda-bld/broken
  Traceback (most recent call last):
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3523, in test
      utils.check_call_env(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 404, in check_call_env
      return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 380, in _func_defaulting_env_to_os_environ
      raise subprocess.CalledProcessError(proc.returncode, _args)
  subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/opt/conda/conda-bld/biobambam_1734549481365/test_tmp/conda_test_runner.sh']' returned non-zero exit status 127.

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 589, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 209, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3727, in build_tree
      test(pkg, config=metadata.config.copy(), stats=stats)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3537, in test
      tests_failed(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3584, in tests_failed
      raise CondaBuildUserError("TESTS FAILED: "  os.path.basename(pkg))
  conda_build.exceptions.CondaBuildUserError: TESTS FAILED: biobambam-2.0.185-h7d28f73_1.tar.bz2
# Last 100 lines of the build log.
category: |-
  compiler error
